label_studio_ml/examples/gliner/model.py

```diff
@@ -0,0 +1,281 @@
+import logging
+import os
+from types import SimpleNamespace
+from typing import List, Dict, Optional
+
+import label_studio_sdk
+import torch
+from gliner import GLiNER
+from label_studio_ml.model import LabelStudioMLBase
+from label_studio_ml.response import ModelResponse
+from label_studio_sdk.objects import PredictionValue
+from tqdm import tqdm
+import json
+from transformers import get_cosine_schedule_with_warmup
+
+logger = logging.getLogger(__name__)
+
+GLINER_MODEL_NAME = os.getenv("GLINER_MODEL_NAME", "urchade/gliner_medium-v2.1")
+logger.info(f"Loading GLINER model {GLINER_MODEL_NAME}")
+MODEL = GLiNER.from_pretrained(GLINER_MODEL_NAME)
+
+
+class GLiNER(LabelStudioMLBase):
```

Comment:
Isn't it in conflict with class name from `gliner` module? Better to rename it to `GLiNERModel` or something. Also, ensure the right imports in `test_api.py`